### PR TITLE
Fix 'dg fyre info get-quota'

### PR DESCRIPTION
**Pull request by jensmueller-com**
This pull request contains the following changes:

- Fix `dg fyre info get-default-sizes` due to a change in one of the FYRE OCP+ APIs
- Fix `dg fyre info get-quick-burn-sizes` due to a change in one of the FYRE OCP+ APIs
- Fix `dg fyre info get-quota` due to a change in one of the FYRE OCP+ APIs

### CODE REVIEW

File: dg/lib/fyre/data/product_group_quota_data.py [hunk 1]
```diff
@@ -52,44 +52,55 @@ def _format_product_group_quota(self, product_group_quota: ProductGroupQuota):
...
         quota_list: List[List[str]] = []
-        quota_list.append(quota_list_element_p)
-        quota_list.append(quota_list_element_x)
-        quota_list.append(quota_list_element_z)
+
+        if "p" in product_group_quota:
+            product_group_quota_p = product_group_quota["p"]
+            quota_list_element_p: List[str] = [
+                "p",
+                product_group_quota_p["cpu"] if "cpu" in product_group_quota_p else "-",
```
Comment (diva1012) [inline dg/lib/fyre/data/product_group_quota_data.py:61]: Does `-` mean default?

File: dg/lib/fyre/data/product_group_quota_data.py [hunk 2]
```diff
@@ -52,44 +52,55 @@ def _format_product_group_quota(self, product_group_quota: ProductGroupQuota):
...
+                str(product_group_quota_p["cpu_percent"]) if "cpu" in product_group_quota_p else "-",
+                product_group_quota_p["disk"] if "disk" in product_group_quota_p else "-",
+                str(product_group_quota_p["disk_percent"]) if "disk" in product_group_quota_p else "-",
+                product_group_quota_p["memory"] if "memory" in product_group_quota_p else "-",
+                str(product_group_quota_p["memory_percent"]) if "memory" in product_group_quota_p else "-",
```
Comment (diva1012) [inline dg/lib/fyre/data/product_group_quota_data.py:66]: This part is repeating three times. Should we extract it as a parametrized method? 

E.g. 
get_product_group_quotaf_for_size(sting size)

then the method uses some key value pairs of name - definition (  "x"-> product_group_quota_x, ...)

**Review by diva1012 — APPROVED** [review]
Thanks, @jensmueller-com , looks good. Maybe you could just add the refactoring for code reuse mentioned in the second comment.

File: dg/lib/fyre/data/product_group_quota_data.py (hunk 1 above)
Comment (jensmueller-com) [reply dg/lib/fyre/data/product_group_quota_data.py]: No, "-" means no quota:

```
$ dg fyre info get-quota
Product group: Db2 Data Gate-Development & QA (Boeblingen) (ID: 420):
Platform    CPU (total)    CPU (used) [%]    Disk (total)    Disk (used) [%]    Memory (total)    Memory (used) [%]
----------  -------------  ----------------  --------------  -----------------  ----------------  -------------------
p           -              -                 -               -                  -                 -
x           512            72                19000           71                 1424              70
z           -              -                 -               -                  -                 -```

**Pull request merged**